ide/docker.ui/src/org/netbeans/modules/docker/ui/wizard/DockerConnectionPanel.java

```diff
@@ -246,14 +258,24 @@ public void validate() throws WizardValidationException {
             }
 
             DockerAction action = new DockerAction(instance);
-            if (!action.ping()) {
+            boolean pingResult = false;
+            
+            try {
+                pingResult = action.pingWithExceptions();
+            } catch (Exception ex) {
+                LOGGER.log(Level.FINE, "docker connection ping failed", ex); // NOI18N
```

Comment:
I would move this to `Level.INFO`. It is worth logging and INFO will not reach the UI, but in the log it is still useful.